charts/redpanda/templates/console/configmap-and-deployment.yaml

```
@@ -15,6 +15,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */}}
 {{ $values := .Values }}
+
+{{/* Secret */}}
+{{ $secretConfig := dict }}
+{{ if and .Values.console.enabled (not .Values.console.secret.create) }}
+{{ $licenseKey := $values.license_key }}
+{{ $secretConfig = merge ( dict
+  "create" true
+  "enterprise" ( dict "license" $licenseKey)
+  ) $secretConfig
+}}
```

NIT: Why merging empty dictionary (from line 20)?

https://github.com/redpanda-data/helm-charts/pull/723/files#diff-0b45ecf3a48bcb79531c1cf2372a743fa93c58bc1ee5193a798e9decd52bf7e4R20

```suggestion
{{ $secretConfig = ( dict
  "create" true
  "enterprise" ( dict "license" $licenseKey)
  )
}}
```

---

I wasnt empty initially, but yes you are right, can remove this safely now.